exercise-1.js

```diff
@@ -296,24 +463,64 @@
 
         function chessCalc(pieces) {
             //your code here
+            let pawn = 1;
+            let rook = 5;
+            let bishop = 3;
+            let knight = 3;
+            let queen = 9;
+            //let king = 'N/A';
+            let sum = 0;
+            for (i = 0; i < pieces.length; i++) {
+                switch (pieces[i]) {
+                    case "pawn":
+                        sum = sum + pawn;
+                        break;
+                    case "rook":
+                        sum = sum + rook;
+                        break;
+                    case "bishop":
+                        sum = sum + bishop;
+                        break;
+                    case "knight":
+                        sum = sum + knight;
+                        break;
+                    case "queen":
+                        sum = sum + queen;
+                        break;
+                    default: 
+                        sum = sum + 0;
```

Comment:
Think about this statement.  Isn't it equivalent to `sum = sum;`?  That statement then sets a variable to itself.  This particular line seems a useless line.  Was there a different intent you had for this line?